hail/python/hail/linalg/blanczos_data_script_new.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
```

Comment:
At some point, remove this whole script, we aren't going to check in test scripts.